config/200-clusterrole.yaml

```diff
@@ -85,6 +85,9 @@ rules:
     resources: ["podsecuritypolicies"]
     resourceNames: ["tekton-triggers"]
     verbs: ["use"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
```

Comment:
Curious if here and below we need all these permissions? Seems like create would suffice because we only emit events?

Reply:
Oh yes only create is required
Thanks for the heads up :+1: